### PR TITLE
Less `<nixpkgs>`

### PR DESCRIPTION
**Pull request by Ericson2314**
I have:

  - [ ] Based work on latest `develop` branch
  - [ ] Followed the [contribution guide](https://github.com/obsidiansystems/obelisk/blob/develop/CONTRIBUTING.md#submitting-changes)
  - [ ] Looked for lint in my changes with `hlint .` (lint found code you did not write can be left alone)
  - [ ] Run the test suite: `$(nix-build -A selftest --no-out-link)`
  - [ ] [Updated the changelog](https://github.com/obsidiansystems/obelisk/blob/develop/CONTRIBUTING.md#in-the-changelog)
  - [ ] (Optional) Run CI tests locally: `nix-build release.nix -A build.x86_64-linux --no-out-link` (or `x86_64-darwin` on macOS)

### CODE REVIEW

File: default.nix [hunk 1]
```diff
@@ -13,8 +13,6 @@ let
   inherit (reflex-platform) hackGet nixpkgs;
   pkgs = nixpkgs;
 
-  inherit (import dep/gitignore.nix { inherit (nixpkgs) lib; }) gitignoreSource;
-
```
Comment (Ericson2314) [inline default.nix]: dead code, I think? Unless this is for downstream consumers.

Comment (cgibbard) [reply default.nix]: If this isn't needed, do we still need the thunk? Is it imported elsewhere?

Comment (Ericson2314) [reply default.nix]: Yes, it is imported elsewhere (can't find in this diff, but I saw it via grepping for `gitignoreSource`)

Comment (Ericson2314) [reply default.nix]: I put it back for now.

**Pull request merged**